src/extensions/__tests__/extension-loader.test.ts

```diff
@@ -125,42 +126,39 @@ describe("ExtensionLoader", () => {
     mockFs.restore();
   });
 
-  it("renderer updates extension after ipc broadcast", async done => {
+  it("renderer updates extension after ipc broadcast", async () => {
     expect(extensionLoader.userExtensions).toMatchInlineSnapshot(`Map {}`);
 
     await extensionLoader.init();
-
-    setTimeout(() => {
-      // Assert the extensions after the extension broadcast event
-      expect(extensionLoader.userExtensions).toMatchInlineSnapshot(`
-        Map {
-          "manifest/path" => Object {
-            "absolutePath": "/test/1",
-            "id": "manifest/path",
-            "isBundled": false,
-            "isEnabled": true,
-            "manifest": Object {
-              "name": "TestExtension",
-              "version": "1.0.0",
-            },
-            "manifestPath": "manifest/path",
+    await delay(10);
```

Comment:
Note: This `delay` (and the old `setTimeout`) is very bad thing. But since you are not adding it, and just changing the way of doing it, it's fine for now.